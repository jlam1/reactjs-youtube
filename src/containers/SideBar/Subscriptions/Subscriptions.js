import React from 'react';
import { Subscription } from './Subscription/Subscription';

class Subscriptions extends React.Component {
	render() {
		return (
			<React.Fragment>
				<Subscription label='Academind' broadcasting />
				<Subscription label='ColdFusion' amountNewVideos={3} />
				<Subscription label='Great Big Story' broadcasting amountNewVideos={10}/>
				<Subscription label='videogamedunkey' amountNewVideos={1}/>
				<Subscription label='PictureFit' />
				<Subscription label='Android Authority' amountNewVideos={2}/>
				<Subscription label='Nintendo' />
			</React.Fragment>
		);
	}
}

export default Subscriptions;