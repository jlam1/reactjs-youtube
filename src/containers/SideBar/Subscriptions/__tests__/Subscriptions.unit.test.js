import React from 'react';
import {shallow} from 'enzyme';
import Subscriptions from '../Subscriptions';

describe('Subscriptions', () => {

	test('Renders', () => {
		const wrapper = shallow(
		  <Subscriptions />
		);
		expect(wrapper).toMatchSnapshot();
	  });

});