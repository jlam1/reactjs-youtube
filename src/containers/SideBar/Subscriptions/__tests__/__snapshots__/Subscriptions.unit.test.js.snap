// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscriptions Renders 1`] = `
<Fragment>
  <Subscription
    broadcasting={true}
    label="Academind"
  />
  <Subscription
    amountNewVideos={3}
    label="ColdFusion"
  />
  <Subscription
    amountNewVideos={10}
    broadcasting={true}
    label="Great Big Story"
  />
  <Subscription
    amountNewVideos={1}
    label="videogamedunkey"
  />
  <Subscription
    label="PictureFit"
  />
  <Subscription
    amountNewVideos={2}
    label="Android Authority"
  />
  <Subscription
    label="Nintendo"
  />
</Fragment>
`;
